int8 left_power
int8 right_power
---
